include/votca/xtp/threecenter.h

```diff
@@ -93,6 +102,8 @@ class TCMatrix_gwbse : public TCMatrix {
 
   void Initialize(int basissize, int mmin, int mmax, int nmin, int nmax);
 
+  void SetGPUStreams(int streams);
```

Comment:
I am not sure about this, I would prefer to have this number be a global like the OPENMPthreads? What do you think?

Reply:
this variable has been removed